File: inputOutput (david&fawaz)/kasirmini.py
nama = input("Masukkan nama barang:")
harga = float(input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah barang: "))

total = harga * jumlah

if total > 24900:
    diskon = total * 0.09
else:
    diskon = 0
total_bayar = total - diskon

print("\n===== Struk Pembayaran =====")
print(f"Nama Barang  : {nama}")
print(f"Harga Barang : {harga}")
print(f"Jumlah Barang: {jumlah}")
print(f"Total Harga  : {total}")
print(f"Diskon       : {diskon}")
print(f"Total Bayar  : {total_bayar}")
print("============================")

File: inputOutput zibril/kasirmini.php
<?php
// Input dari user
$nama = readline("Masukkan nama barang: ");
$harga = floatval(readline("Masukkan harga barang: "));
$jumlah = intval(readline("Masukkan jumlah barang: "));

// Perhitungan
$total = $harga * $jumlah;

if ($total > 56000) {
    $diskon = $total * 0.06;
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

// Output struk
echo "\n===== Struk Pembayaran =====\n";
echo "Nama Barang   : $nama\n";
echo "Harga Barang  : $harga\n";
echo "Jumlah Barang : $jumlah\n";
echo "Total Harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total Bayar   : $total_bayar\n";
echo "============================\n";
?>
